src/test/regress/expected/multi_shard_update_delete_0.out

```diff
@@ -766,7 +775,7 @@ DECLARE test_cursor CURSOR FOR SELECT * FROM users_test_table;
 FETCH test_cursor;
  user_id | value_1 | value_2 | value_3 
 ---------+---------+---------+---------
-       8 |       4 |      13 |       0
+       8 |       4 |       0 |       0
```

Comment:
seems like this would be 1 about half the time (postgres casts double to int using rounding)